MekHQ/src/mekhq/gui/dialog/DataLoadingDialog.java

```diff
@@ -132,6 +133,11 @@ public Void doInBackground() {
             } catch (Exception ex) {
     			ex.printStackTrace();
             }
+            try{
+                CurrencyManager.getInstance().loadCurrencies();
+            } catch (Exception ex) {
+                ex.printStackTrace();;
```

Comment:
In new code use `MekHQ.getLogger().error(...)` for these (also a doubled up semi-colon).